lib/nerves_network/dhcp_manager.ex

```diff
@@ -146,8 +147,9 @@ defmodule Nerves.Network.DHCPManager do
 
   ## covers STOP, RELEASE, FAIL and EXPIRE in Nerves.Network.Dhclientv4
   defp consume(:dhcp, :ifdown, state), do: consume(:dhcp, {:ifdown, :no_info}, state)
+
   defp consume(:dhcp, {:ifdown, info}, state) do
-    Logger.debug fn -> "#{__MODULE__}: consume (context = :dhcp) :ifup info: #{inspect info}" end
+    Logger.debug fn -> "#{__MODULE__}: consume (context = :dhcp) :ifdown info: #{inspect info}" end
```

Comment:
Minor nit-pick:  We don't need to wrap in `fn -> end` anymore as that was fixed in the latest Elixir.  Also __MODULE__ is printed out by the Logger module format string now too

Reply:
Yup this is the residue from the previous implementation

Reply:
went through the unit so it now neither uses the fn -> nor \#{__MODULE__}